build.sbt

```diff
@@ -45,6 +69,15 @@ def test(scope: String = "test, it"): Seq[ModuleID] = Seq(
   "com.github.tomakehurst" % "wiremock" % "2.6.0" % scope
 )
 
+def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] = tests map {
+  test =>
+    Group(
+      test.name,
+      Seq(test),
+      SubProcess(ForkOptions(runJVMOptions = Seq("-Dtest.name=" + test.name, "-Dlogger.resource=logback-test.xml")))
+    )
+}
+
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin) ++ plugins : _*)
   .settings(scalaSettings: _*)
```

Comment:
You need to add `.settings(coverageSettings: _*)` and `.settings(playSettings: _*)` here in order to apply your new settings